### PR TITLE
Prevent error from showing in plot on halt

### PR DESCRIPTION
**Pull request by qiemem**

### CODE REVIEW

**Comment by mrerrormessage** [comment]
So I think this approach is generally sound, but `plot` isn't allowed to depend on `nvm`, which contains `HaltException`. So we might have to rethink this a little. I'll comment again tomorrow if I have any good ideas.

**Comment by qiemem** [comment]
Moving `HaltException` to `api` seems like a pretty reasonable thing. I'm a little surprised it wasn't there already.

**Comment by qiemem** [comment]
Build log says

> The job exceeded the maximum time limit for jobs, and has been terminated.

File: netlogo-core/src/main/api/HaltSignal.scala [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package org.nlogo.api
```
Comment (mrerrormessage) [inline netlogo-core/src/main/api/HaltSignal.scala]: 🤖 requires copyright notice 🤖

**Comment by mrerrormessage** [comment]
Just needs a copyright notice in one file. Other than that it should be good. The push build failed with a timeout, but the PR build passed, so I'm going to chalk the failure up to Travis flakiness.

**Pull request merged**